Doc/library/ctypes.rst

```diff
@@ -1408,6 +1408,10 @@ accessing the function through an attribute caches the result and therefore
 accessing it repeatedly returns the same object each time.  On the other hand,
 accessing it through an index returns a new object each time:
 
+.. doctest::
```

Comment:
I removed the doctest directive. However, there is one point for taking it: in some editors, if you use the `::` before the example you enable the syntax hightligh for the example, but the example will not be doctested. If you use just the `:`, you lose the syntax highligh but the example will be doctested. If you eventually use the `doctest` directive, you have both syntax highlight and doctest.